openapi.yaml

```diff
@@ -269,10 +269,13 @@ paths:
           content:
             application/json:
               schema:
-                enum:
-                - {}
+                properties:
```

Comment:
Here we must send back the name of the deployment, the user will need it if it's generated by us

Reply:
API accepts a name in the request for each `deployment`. The `name` property is optional:
https://api.athenian.co/v1/ui/#/events/notify_deployments
```
POST /v1/events/deployments
[
  {"name": "my-deployment-00", "components": [...]}
]
```
So if a unique name is computed when sending the notification it's possible to reference the same name in a following request to set the labels.
Does this satisfy your use case ?

Reply:
Yes, that's perfect